generators/app/templates/src/controller/BaseController.ts

```diff
@@ -0,0 +1,68 @@
+import Controller from "sap/ui/core/mvc/Controller";
+import History from "sap/ui/core/History";
+import UIComponent from "sap/ui/core/UIComponent";
+import Model from "sap/ui/model/Model";
+import ResourceBundle from "sap/ui/model/resource/ResourceModel";
+import View from "sap/ui/mvc/View";
+import Router from "sap/ui/core/routing/Router";
+
+/**
+ * @namespace <%= appId %>.controller
+ */
+export default class BaseController extends Controller {
+
+	/**
+	 * Convenience method for getting the view model by name in every controller of the application.
+	 * @public
+	 * @param {string} sName? the model name
```

Comment:
please don‘t type the input params in jsdoc - you casted them already in the code, let TS take care of the rest :)
applies to the other jsdoc blocks as well